### DISCUSSION TITLE
APIRouter: unexpected keyword argument 'prefix'

### DISCUSSION THREAD

**Discussion by jintolonappan** [fastapi/fastapi#6995] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Issue

I was following the official tutorial related to [bigger applications](https://fastapi.tiangolo.com/tutorial/bigger-applications/) and came across a `TypeError` with all the parameters to `APIRouter()`

Examples:
```py
TypeError: __init__() got an unexpected keyword argument 'prefix'
TypeError: __init__() got an unexpected keyword argument 'tags'
TypeError: __init__() got an unexpected keyword argument 'responses'
```

Code used:
```py
router = APIRouter(
    prefix="/items",
    tags=["items"],
    dependencies=[Depends(get_token_header)],
    responses={404: {"description": "Not found"}},
)
```

However, I noticed that each of the argument ***works*** if I use them in `main.py` as:

```py
app.include_router(
    admin.router,
    prefix="/admin",
    tags=["admin"],
    dependencies=[Depends(get_token_header)],
    responses={418: {"description": "I'm a teapot"}},
)
```

Is this an issue in documentation or code?

### Environment

* OS: macOS Catalina 10.15.7
* FastAPI Version: 0.61.1
* Python version: Python 3.8.5

**Comment by Mause** [comment, marked as answer]
You are looking at documentation for the current version, 0.63.0. Please update if you wish to use those features

**Comment by jintolonappan** [comment]
Thanks @Mause - It worked.

**Comment by tiangolo** [comment]
Thanks for the help here @Mause ! 👏  🙇 

Thanks for reporting back and closing the issue @jintolonappan  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
